### ISSUE TITLE
Internet Error: -4

### ISSUE THREAD

**Issue by agguser** [benibela/xidel#15]
```
% xidel https://www.archlinux.org/packages/extra/x86_64/firefox/ -e //title
Error:
Internet Error: -4 
when talking to: https://www.archlinux.org/packages/extra/x86_64/firefox/
```

**Comment by benibela** [comment]
That is probably a failed HTTPS connection.

Are you using Windows? On some Windows versions you need to enable TLS1.2: https://blogs.msdn.microsoft.com/kaushal/2011/10/02/support-for-ssltls-protocols-on-windows/

Try to open the page in the Internet Explorer first

**Comment by agguser** [comment]
I use Arch Linux.

**Comment by benibela** [comment]
Then it seems it does not support TLS1.2 on Linux.

Xidel uses Synapse, there is a [patch](https://sourceforge.net/p/synalist/mailman/message/35785147/) that seems to fix it, if you want to compile it yourself

**Comment by agguser** [comment]
A workaround:

`curl https://www.archlinux.org/packages/extra/x86_64/firefox/ | xidel -s - -e //title`

**Comment by benibela** [comment]
Seems they have merged the openssl 1.1 patch in trunk: https://sourceforge.net/p/synalist/code/206/ but not made a new release

And I have rebuilt the xidel binaries with that trunk https://sourceforge.net/projects/videlibri/files/Xidel/Xidel%20development/
